python/lsst/daf/persistence/policy.py

```diff
@@ -376,6 +376,36 @@ def getStringArray(self, key):
             val = [val]
         return val
 
+    def __lt__(self, other):
```

Comment:
it had to do with putting these into a container, these rich comparison methods were needed for that.